docs/run/primitives.mdx

```diff
@@ -76,7 +168,7 @@ for each primitive type:
 
 1. `Sampler` and `Estimator`
```

Comment:
V2 reference implementation are called `StatevectorSampler` and `StatevectorEstimator`